cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
It seems like cypress just keeps making these files if they don't exist so even though it has nothing I decided to keep them.

Reply:
Good to know! 👍